.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+| Q                 | A
+| ----------------- | ----------
+| Bug fix?          | yes/no
+| New feature?      | yes/no    <!-- please update the /CHANGELOG.md file -->
+| BC breaks?        | no     
+| Related Issue     | Fix #...  <!-- will close issue automatically, if any -->
+| Need Doc update   | yes/no
+
+
+## What was changed
```

Comment:
We should either use `What has changed` or `What was changed?` (with the question mark) if I'm not wrong. cc @codeharmonics